web-bundle/src/main/java/com/graphhopper/http/TruckAverageSpeedParser.java

```diff
@@ -233,7 +238,19 @@ boolean isCarLike() {
     public void handleWayTags(int edgeId, EdgeIntAccess edgeIntAccess, ReaderWay way) {
         super.handleWayTags(edgeId, edgeIntAccess, way);
         if (getCustomSpeedIdType(idToCustomSpeeds) == CustomSpeedId.IdType.SEID) {
-            throw new NotImplementedException("SEID-based custom speeds are not yet implemented for trucks!");
+            ImmutableMap<String, Double> stableEdgeIdToMaxSpeed = getStableEdgeIdBasedCustomSpeeds(idToCustomSpeeds);
```

Comment:
Note: `super.handleWayTags` here will handle the setting of speeds without custom speeds in mind, so in cases where this new logic doesn't call `setSpeed` explicitly, rest assured a speed is already present there